spec/system/schools/estimated_annual_consumptions_spec.rb

```diff
@@ -85,6 +85,10 @@
         latest_estimate = school.latest_annual_estimate
         expect(latest_estimate.electricity).to eq 6000.0
       end
+
+      it 'does not show delete link' do
+        expect(page).to_not have_content("Delete")
```

Comment:
Would it be more expressive to use "..to_not have_link.." rather than "..to_not have_content.." ?